cmd/describer.go

```diff
@@ -181,7 +181,7 @@ func toYesNo(tf bool) string {
 func (e *ExportsDescriber) Describe() string {
 	table := tablewriter.CreateTable()
 	table.AddTitle("Exports")
-	table.AddHeaders("Name", "Type", "Subject", "Public", "Revocations", "Tracking", "Description", "Info URL")
```

Comment:
why delete the current 'user formatted' info?

Reply:
I updated to make a second table